### PR TITLE
feat(cambion): add state to be consistent

### PR DESCRIPTION
**Pull request by Neppkun**
### What did you fix? 
Changed line 21 in lib/CambionCycle.js to `this.state` from `this.active`

---

### Evidence/screenshot/link to line

See [line 21](https://github.com/WFCD/warframe-worldstate-parser/compare/master...Neppkun:warframe-worldstate-parser:master?expand=1#diff-a55ae305eee93976388a02c2a0bfa88797fc42e68bb96ceec43c4a74f6b48440R21)

Here's a screenshot of my terminal using it as well: 
image

### Considerations
- Does this contain a new dependency? **No**
- Does this introduce opinionated data formatting or manual data entry? **Yes**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **Yes(?)**
- Have I run the linter? **No**
- Is is a bug fix, feature request, or enhancement? **Feature Request**

### CODE REVIEW

**Review by TobiTenno — CHANGES_REQUESTED** [review]
Not sure what you did but it looks like you rewrote the whole file. Check if you changed line endings or similar.

**Comment by Neppkun** [comment]
> Not sure what you did but it looks like you rewrote the whole file. Check if you changed line endings or similar.
> 
> Additionally, for passivity, the state should be new and the same value as active, not just a name change.

Added back the old key. And I have no idea why Github says the entire file is overwritten, but it doesn't seem to be a problem image

**Comment by TobiTenno** [comment]
if you're on windows and you used notepad, it changed all of the line endings, so please start over and either do it from CLI, or do it from notepad++/vscode/atom/etc. and leave the line endings as LF

**Comment by Neppkun** [comment]
> if you're on windows and you used notepad, it changed all of the line endings, so please start over and either do it from CLI, or do it from notepad++/vscode/atom/etc. and leave the line endings as LF

@TobiTenno oh, just noticed I was using CRLF. My bad, sorry. Changed it back to LF and it seems to be good now.

**Comment by TobiTenno** [comment]
still showing all lines as CRLF

**Comment by Neppkun** [comment]
> still showing all lines as CRLF

@TobiTenno Are you sure you're checking for repo lines and not working directory?

return of `git ls-files --eol` (where `i/[type]` is repository) shows LF
image

opening the repo in github.dev also shows LF
image

**Comment by TobiTenno** [comment]
i'm looking at the lint job and the diff on github. @Neppkun

File: lib/CambionCycle.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = class CambionCycle extends WorldstateObject {
 
     ({ activation: this.activation, expiry: this.expiry, timeLeft: this.timeLeft } = cetusCycle);
 
+    this.state = cetusCycle.isDay ? 'fass' : 'vome';
     this.active = cetusCycle.isDay ? 'fass' : 'vome';
```
Comment (TobiTenno) [inline lib/CambionCycle.js:22]:
```suggestion
    this.state = cetusCycle.isDay ? 'fass' : 'vome';
    this.active = this.state;
```

**Comment by Neppkun** [comment]
Great then :D

**Comment by TobiTenno** [comment]
oh, just noticed you didn't add unit tests... sorry, need to just add one to verify that new field

**Comment by Neppkun** [comment]
I'm afraid I've got no clue how unit testing works. And besides, do you really need unit testing on one line of code that's just the previous line with a different name?

**Comment by TobiTenno** [comment]
> I'm afraid I've got no clue how unit testing works. And besides, do you really need unit testing on one line of code that's just the previous line with a different name?

yeah, that's kinda the point of unit testing... to catch if something happens to that field. new name, etc.

just search for cambion tests... https://github.com/WFCD/warframe-worldstate-parser/blob/master/test/unit/cambion.spec.js#L65 add a copy of this line, then you're done

**Comment by Neppkun** [comment]
> yeah, that's kinda the point of unit testing... to catch if something happens to that field. new name, etc.

Alright then, my bad. I just never  worked with unit tests so didn't know.

> just search for cambion tests... https://github.com/WFCD/warframe-worldstate-parser/blob/master/test/unit/cambion.spec.js#L65 add a copy of this line, then you're done

I believe I [did it](https://github.com/WFCD/warframe-worldstate-parser/pull/427/commits/9454ffb678a5ee9f747048c9b5bb30dd09c7cc94)

**Comment by TobiTenno** [comment]
The unit test is fine, but I requested a logic change to reduce dupe code and you resolved it without response.

File: lib/CambionCycle.js (hunk 1 above)
Comment (TobiTenno) [reply lib/CambionCycle.js:22]: Please don't resolve if you both don't change and don't reply.

**Comment by Neppkun** [comment]
> The unit test is fine, but I requested a logic change to reduce dupe code and you resolved it without response.

oh, I thought resolving applied the requested changes. My bad.

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**